include/SW_Domain.h

```diff
@@ -0,0 +1,25 @@
+#ifndef SWDOMAIN_H
+#define SWDOMAIN_H
+
+#include "include/SW_datastructs.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* =================================================== */
+/*                  Local Definitions                  */
+/* --------------------------------------------------- */
+
+#define NUM_DOM_IN_KEYS 8 // Number of possible keys within `domain.in`
+#define KEY_NOT_FOUND -1 // A key within `domain.in` is not a valid one
+
+void SW_DOM_read(char *InFiles[], SW_DOMAIN* SW_Domain, LOG_INFO* LogInfo);
+void SW_DOM_setModelTime(SW_MODEL *SW_Model, SW_DOMAIN *SW_Domain);
+int domain_inkey_to_id(char *key);
```

Comment:
do `domain_inkey_to_id()`, `NUM_DOM_IN_KEYS`, and `KEY_NOT_FOUND` need to be exported - it seems they need to be local, i.e., static, function/defines?

Reply:
I agree with `domain_inkey_to_id()` and `NUM_DOM_IN_KEYS`. But `KEY_NOT_FOUND` is used by `SW_NC_read_files()`. But honestly, that can be changed to a local define as well because `SW_Domain.h` is included in `SW_netCDF.h` for that reason.

Reply:
Since `KEY_NOT_FOUND` is going to be used in different modules, maybe we should place it in "SW_Defines.h" so that it's accessible

Reply:
I agree